config.style

```diff
@@ -270,6 +270,15 @@ Layer {
 	SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/ps/4.0:ProtectedSite
 	Styles: PS_ProtectedSite
 }
+Layer {
+	id: "PS_ProtectedSite"
```

Comment:
Not sure if there will be issues with two layers having the same id. But TBH I have no idea how this repository works. Just in case the id is used for any linkage etc., it might become an issue when the same id is used twice.

Reply:
@JohannaOtt That can be tested once this PR is live.

Reply:
@stempler Do you know if it is possible to have two layers with the same id? (PS_ProtectedSite)?